statsmodels/sandbox/stats/diagnostic.py

```diff
@@ -239,7 +239,7 @@ def acorr_ljungbox(x, lags=None, boxpierce=False):
         If lags is a list or array, then all lags are included up to the largest
         lag in the list, however only the tests for the lags in the list are
         reported.
-        If lags is None, then the default maxlag is 12*(nobs/100)^{1/4}
```

Comment:
The 12*(nobs/100)^(1/4) expression shows up in a handful of places (and with slightly differing round/ceil/floor details) without super-clear reasoning as to why.  Not an issue for this PR, but it would be nice to clarify+unify+document that.